ui/page/receive_page.go

```diff
@@ -345,7 +345,7 @@ func (pg *ReceivePage) Handle() {
 		info := modal.NewInfoModal(pg.Load).
 			Title("Receive DCR").
 			Body("Each time you receive a payment, a new address is generated to protect your privacy.").
-			PositiveButton("Got it", func() {})
+			NegativeButton("Got it", func() {})
```

Comment:
Why is change done in multiple places?

Reply:
The `NegativeButton` is designed to minimize modals when no specific action is to be carried out.